src/sentry/api/endpoints/team_projects.py

```diff
@@ -93,24 +100,35 @@ def get(self, request: Request, team) -> Response:
             paginator_cls=OffsetPaginator,
         )
 
+    @extend_schema(
+        # Ensure POST is in the projects tab
+        tags=["Projects"],
+        operation_id="Create a New Project",
+        parameters=[
+            GLOBAL_PARAMS.ORG_SLUG,
+            GLOBAL_PARAMS.TEAM_SLUG,
+            GLOBAL_PARAMS.NAME("The name of the project.", required=True),
+            GLOBAL_PARAMS.SLUG(
+                "Optional slug for the project. If not provided a slug is generated from the name."
+            ),
```

Comment:
this is currently confusing as `NAME` is deprecated across the app, so only slug is used. Name is used in SCIM for compatibility, but it doesn't show anywhere in the UI. this may be confusing, but i guess it is just the state of things for now. not sure if there's any action to be taken here but I want to note that. cc @evanpurkhiser

Reply:
I'm planning to change those descriptions in a later PR. Part of the plan for cleaning up the APIs is to consolidate the use of name+slug for both teams and projects. I've noted this problem [here on Notion](https://www.notion.so/sentry/Things-to-Deprecate-fea325af798c4870a3a788e60470ac0c?pvs=4#a74cc50319144b6e801ba0729df27ce2)